influxql/parser_test.go

```diff
@@ -1244,6 +1244,9 @@ func TestParser_ParseStatement(t *testing.T) {
 		{s: `select derivative() from myseries`, err: `invalid number of arguments for derivative, expected at least 1 but no more than 2, got 0`},
 		{s: `select derivative(mean(value), 1h, 3) from myseries`, err: `invalid number of arguments for derivative, expected at least 1 but no more than 2, got 3`},
 		{s: `SELECT field1 from myseries WHERE host =~ 'asd' LIMIT 1`, err: `found asd, expected regex at line 1, char 42`},
+		{s: `SELECT value > 2 FROM cpu`, err: `invalid operator for field > at line 1, char 8`},
```

Comment:
would be better to have the actual name in the error. Also mention that it's an invalid operator for the select clause. Maybe something like:

```
invalid operator in select clause: > at line 1, char 8. these belong in the where clause
```